### ISSUE TITLE
Where is the downloaded file?

### ISSUE THREAD

**Issue by sinanad** [TwitchRecover/TwitchRecover#347]
It said the vod was downloaded successfully but where? At no point in the command line is this information mentioned??

**Comment by MrIronstone** [comment]
If you haven't specified any path, it automatically downloads to the user directory of the user you are logged in to. It is like C:\Users\[your account's user name]

**Comment by Perimedion** [comment]
I entered "desktop" as destination. 
Any idea where it might have gone?

**Comment by ghost** [comment]
@Perimedion Enter an absolute path.....  C:\Users\YOUR_USERNAME\Desktop

It probably didn't write anywhere because it couldn't find the path.

desktop isn't a path to the application its just the word "desktop".

**Comment by Perimedion** [comment]
@ItIckeYd yeah, I was really expecting it to throw an error when I just entered "desktop", but then it actually proceeded to download, or at least it seemed like it did, but I cannot find the file now. 
Guess I'll just download again.
